packages/react-components/src/components/VideoGallery.test.tsx

```diff
@@ -198,7 +201,7 @@ describe('VideoGallery floating local video layout tests', () => {
     ).toBe(false);
   });
 
-  test('should render all remote video tiles in the grid', () => {
+  test('should render all remote video tiles in the grid and others to horizontal gallery', () => {
```

Comment:
Why was this test name updated? was there always going to be a Horizontal gallery in this test?

Reply:
Just to be more specifc to a horizontal gallery since I have now added vertical gallery tests